LogisticApp/ViewModel/Forms/EmployeeCreateFormViewModel.cs

```diff
@@ -98,11 +98,24 @@ public override void loadData(BaseEntity entity)
             BirthDate = employee.birthDate;
         }
 
-        //TODO add validation if there will be enough time
         public override void save()
         {
             Creator.createOrUpdate("employee");
         }
 
+        public override bool canSave()
+        {
+            if (_firstname == "" ||
+                _lastname == "" ||
+                _hourlyPayment < 0 ||
+                DateTime.Now - _birthDate < TimeSpan.FromDays(5840) ||
```

Comment:
This number of days is some precise date? Won't it be better to check DateTime.Today.Addyears(-number_of_years)?

Reply:
yup, this is exactly 1st of january 2000

Reply:
> Won't it be better to check DateTime.Today.Addyears(-number_of_years)?
How about this?